src/_guides/language/evolution.md

```diff
@@ -313,74 +375,75 @@ multiple versions of the Dart language.
 The compiler determines what version the code is targeting,
 and it interprets the code according to that version.
 
-Language versioning is important on the rare occasions
-when Dart introduces an incompatible feature like [null safety][].
-Code that used to compile cleanly before null safety
-(but perhaps crash at runtime)
-might no longer compile once null safety is enabled.
-Because migrating your apps and packages—and all the packages they depend on—to
-null safety might take a while,
-Dart uses language versioning to support
-using non-null-safe code alongside null-safe code.
-For an example of how an app or package can migrate to
-a new language version
-with an incompatible feature (null safety, for example), see 
+Language versioning becomes important on the rare occasions when Dart
+introduces an incompatible feature like [null safety][].
+When Dart introduces a breaking change, code that
+did compile might no longer compile.
+Language versioning allows you to set each library's language version
+to maintain compatibility.
+
+In the case of null safety, Dart SDKs 2.12 through 2.19 allowed you
```

Comment:
This general area of text has got to a great point. Thanks for all of the back and forth with me here!